channelserver/src/server.rs

```diff
@@ -63,6 +64,8 @@ pub struct ClientMessage {
     pub channel: Uuid,
     /// Sender info
     pub sender: meta::SenderData,
+    /// Abuse reporting system
+    pub abuser: Option<ip_rate_limit::IPReputation>,
```

Comment:
The name `abuser` doesn't help me understand what this field is for - I spent a bit of time trying to figure out which IP was supposed to be "the abuser" before I realized it was the IPRep client itself.  Probably should have been obvious from the type, but still...maybe `iprep_client` or similar would be a more helpful name.

Reply:
Agreed. I thought I had switched that to `ip_rep`. My apologies.